easybuild/easyconfigs/c/CMake/CMake-3.10.3-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.10.3'
```

Comment:
bump to most recent version `3.11.4`?

Reply:
We should do a last update, and then try and stick to `3.11.4` for the `2018b` generation of easyconfigs...

Reply:
Done the 3.11.4 upgrade etc.